src/components/Section.js

```diff
@@ -56,7 +56,8 @@ class Section extends Component {
         paddingTop: mjAttribute('padding-top'),
         paddingBottom: mjAttribute('padding-bottom'),
         paddingRight: mjAttribute('padding-right'),
-        paddingLeft: mjAttribute('padding-left')
+        paddingLeft: mjAttribute('padding-left'),
+        padding: mjAttribute('padding')
```

Comment:
I haven't seen that `MJMLColumnElement` has already `padding` thanks for noticing ! Yes it would be nice to !